central/main.go

```diff
@@ -245,6 +245,15 @@ func main() {
 	devmode.StartOnDevBuilds("central")
 
 	log.Infof("Running StackRox Version: %s", pkgVersion.GetMainVersion())
+	// TODO: ROX-12750 update with new list of replaced/deprecated resources
+	log.Warn("Next access resources will be replaced:\n" +
+		"	Access replaces AuthProvider, Group, Licenses and User\n" +
+		"	DeploymentExtension replaces Indicator, NetworkBaseline, ProcessWhitelist and Risk\n" +
+		"	Integration replaces APIToken, BackupPlugins, ImageIntegration, Notifier and SignatureIntegration\n" +
+		"	Image now also covers ImageComponent\n" +
+		"The resources getting deprecated, but not replaced yet are:\n" +
+		"	Administration will replace AllComments, Config, DebugLogs, NetworkGraphConfig, ProbeUpload, ScannerDefinitions, SensorUpgradeConfig and ServiceIdentity.\n" +
+		"	Compliance will replace ComplianceRuns")
```

Comment:
ClusterCVE is missing, also in the UI banner.

Reply:
See comment below - will `ClusterCVE` be replaced by other resource or just removed?

Reply:
`ClusterCVE` -> `Cluster`